src/utils/daos.c

```diff
@@ -88,6 +88,8 @@ cont_op_parse(const char *str)
 		return CONT_UPDATE_ACL;
 	else if (strcmp(str, "delete-acl") == 0)
 		return CONT_DELETE_ACL;
+	else if (strcmp(str, "set-owner") == 0)
```

Comment:
I think help_hdlr() function will need a line of output showing this new command in the usage print

Reply:
Good catch, will fix